src/components/Banner/Banner.tsx

```diff
@@ -1,6 +1,6 @@
 import React from 'react';
 
-const Banner = () => {
+export default function Banner() {
```

Comment:
@antosan is this a linter based change or a stylistic preference?
(the function declaration)

Reply:
@orama254 This is not part of the linter. I checked the components in the project and the majority were implemented as function declarations. So while adding types for the props across the project, I opted to make them uniform.

Reply:
noted.